pupil-spa/src/app/check/check.component.ts

```diff
@@ -149,7 +153,16 @@ export class CheckComponent implements OnInit {
         this.isWarmUp = true;
         // console.log(`state: ${stateDesc}: question is ${matches[ 1 ]}`);
         this.question = this.warmupQuestionService.getQuestion(parseInt(matches[ 1 ], 10));
-        this.viewState = 'question';
+        this.viewState = 'practice-question';
+        break;
+      }
+      case CheckComponent.spokenWarmupQuestionRe.test(stateDesc): {
+        // Show the warmup question screen
+        const matches = CheckComponent.spokenWarmupQuestionRe.exec(stateDesc);
+        this.isWarmUp = true;
+        // console.log(`state: ${stateDesc}: question is ${matches[ 1 ]}`);
```

Comment:
do you still need that comment?

Reply:
Yes, will convert these console.log to logger entries at some point soon.